cli/Makefile

```diff
@@ -9,6 +9,13 @@ env:
 test: env
 	bin/test.sh
 
+binary: clean env packages
+	rm -rf dist build
```

Comment:
Let's add this to the clean step/target. We can add it to the `bin/clean.sh` script.